### PR TITLE
Admin dashboard

### PR DESCRIPTION
**Pull request by mjvalade**
Added admin-dashboard with hardcoded admin auth.
Route admin only to admin-dashboard to show user results.
Added date picker to admin-dashboard, console logging the selected dates. Need to finish function to grab dates and render only results from those dates.
Replaced calendar page/route with app directions to render user directions.
Some styling of general pages and admin-dashboard page.

### CODE REVIEW

File: app/adapters/application.js [hunk 1]
```diff
@@ -3,8 +3,8 @@ import { Adapter } from 'ember-pouch';
 
 PouchDB.debug.enable('*');
 
-var remote = new PouchDB('https://ab255.cloudant.com/wellness');
-var db = new PouchDB('local_pouch');
+let remote = new PouchDB('https://ab255.cloudant.com/wellness');
```
Comment (madison-codes) [inline app/adapters/application.js:6]: Love the refactoring!

File: app/components/admin-dashboard.js [hunk 2]
```diff
@@ -1,4 +1,15 @@
...
 
 export default Ember.Component.extend({
+  actions: {
+    searchDates: function() {
+      let dStart = this.get('dateStart');
+      let dEnd = this.get('dateEnd');
+      let start = Date.parse(dStart);
+      let end = Date.parse(dEnd);
+
+      console.log("dateStart:", dStart, start);
```
Comment (madison-codes) [inline app/components/admin-dashboard.js:11]: lets remember to take these out - I know I asked you to push up what you had, it is just more of a reminder for myself :)

File: app/routes/application.js [hunk 3]
```diff
@@ -4,16 +4,16 @@ import ApplicationRouteMixin from 'ember-simple-auth/mixins/application-route-mi
 export default Ember.Route.extend(ApplicationRouteMixin, {
 
   actions: {
-    login () {
-      var lockOptions = {
+    login: function() {
```
Comment (madison-codes) [inline app/routes/application.js:7]: Do you prefer this way of writing a function within your actions/ routes/ components? I have no preference and am sure I am switching back and forth. From this point on should I stick with this format?

File: package.json [hunk 4]
```diff
@@ -25,6 +25,7 @@
     "ember-cli": "2.8.0",
     "ember-cli-app-version": "^1.0.0",
     "ember-cli-babel": "^5.1.6",
+    "ember-cli-datepicker": "2.0.1",
```
Comment (madison-codes) [inline package.json:28]: Does this cli allow us to access the user's data? whatever they choose for their date range?

File: public/assets/info.svg [hunk 5]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<?xml version="1.0" ?><!DOCTYPE svg  PUBLIC '-//W3C//DTD SVG 1.1//EN'  'http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd'><svg enable-background="new 0 0 39 115" height="115px" id="Layer_1" version="1.1" viewBox="0 0 39 115" width="39px" xml:space="preserve" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g><path d="M26.102,91.63l3.372-13.589l7.465-30.088c1.33-5.36-1.938-10.783-7.298-12.113l-0.971-0.241   c-0.652-0.162-1.303-0.252-1.949-0.283l0.008-0.199c0,0-0.161,0.072-0.437,0.197c-3.072-0.011-7.561,2.97-7.561,2.97   c-8.72,6.079-20.654,14.141-18.886,17.981c2.678,5.816,15.748-9.487,12.618,3.131L9.091,72.984l-7.465,30.088   c-1.33,5.361,1.504,10.676,6.864,12.006l0.971,0.241c0.652,0.161,1.303,0.253,1.95,0.282l-0.009,0.199c0,0,0.162-0.071,0.438-0.197   c3.072,0.012,7.561-2.97,7.561-2.97c8.72-6.079,20.871-14.087,19.103-17.928C35.824,88.89,22.972,104.247,26.102,91.63z"/><circle cx="24.727" cy="15.217" r="14"/></g></svg>
```
Comment (madison-codes) [inline public/assets/info.svg:1]: really love this new info icon :)

**Review by madison-codes — APPROVED** [review]

File: app/routes/application.js (hunk 3 above)
Comment (mjvalade) [reply app/routes/application.js:7]: I do but that's also what I saw most often so just trying to stay consistent.

File: package.json (hunk 4 above)
Comment (mjvalade) [reply package.json:28]: I don't think so. I think just admin can set what they are seeing on the screen. But I realized if we're saving the date as only "Mon" in the database we can't actually use this to search for dates b/c it needs an actual date.

File: public/assets/info.svg (hunk 5 above)
Comment (mjvalade) [reply public/assets/info.svg:1]: 👍

**Pull request merged**